enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -151,7 +137,6 @@ export const InteractiveQuestionProvider = ({
     mode,
     onSave: handleSave,
     onCreate: handleCreate,
-    modelsFilterList: mapEntityTypeFilterToDataPickerModels(entityTypeFilter),
```

Comment:
this is used in the modern data picker. We don't use that anymore as embedding has its own data pickers now.